docs/getting-started.md

```diff
@@ -67,6 +67,10 @@ kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/bu
 If you want to add Windows Nodes to your cluster, please refer to the
 installation instructions in [windows.md](/docs/windows.md).
 
+If you want to deploy Antrea on a cluster with IPv6 Service CIDR, please
+fill in the `ServiceCIDRv6` field in `antrea.yml` and uncomment it before
```

Comment:
Not sure if this is the good place for this instruction since we do have some other requirements for other modes but they are listed in their own section/doc. Do we have a doc about how to enable IPv6/dual stack in Antrea?

Reply:
No. And I don't think there are many things to talk about for IPv6? @mengdie-song

Reply:
Now that AntreaProxy is enabled by default, do we still require `ServiceCIDRv6`?

Reply:
Yes, then we can delete these `serviceCIDR` fields.

Reply:
Deleted these lines.

Reply:
we shouldn't delete the fields as AntreaProxy can be disabled, but yes we don't need to document the fields here

Reply:
There is no feature gate for dual stack in Antrea, but we may want to document how to initialize a dual stack Kubernetes cluster via `kubeadm init` with IPv6DualStack feature gate. Also, there are some openflow pipeline changes to support IPv6 and we may want to enhance document for this part.